certbot/tests/account_test.py

```diff
@@ -98,14 +93,13 @@ def test_it(self):
         self.assertEqual([account], self.storage.find_all())
 
 
-class AccountFileStorageTest(TempDirTestCase):
+class AccountFileStorageTest(test_util.ConfigTestCase):
     """Tests for certbot.account.AccountFileStorage."""
 
     def setUp(self):
         super(AccountFileStorageTest, self).setUp()
 
-        self.config = mock.MagicMock(
-            accounts_dir=os.path.join(self.tempdir, "accounts"))
+        self.config.accounts_dir = os.path.join(self.tempdir, "accounts")
```

Comment:
We actually don't need this line as `NamespaceConfig` defines `accounts_dir` relative to `config_dir` ([source](https://github.com/certbot/certbot/blob/edcfc49303d4ddd2e69ba6ac6933af01d00e332c/certbot/configuration.py#L67)).